tests/Resources/modules/translationtest/views/templates/admin/someFile.with-things.tpl

```diff
@@ -0,0 +1 @@
+<h1>{l s='Smarty template' mod='translationtest'}</h1>
```

Comment:
Is it just for test purposes?

Reply:
yes, this is the "translationtest" module, it's only for tests @LouiseBonnard

Reply:
Okay, so it won't be part of the Crowdin catalog? I just want to be sure. :-)

Reply:
Nope it shouldn't...  👍